ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -325,25 +337,21 @@
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "DEFENDANT_RESPONSE_SPEC",
-    "CaseFieldID": "specClaimResponseTimelineLabel",
-    "PageLabel": " ",
+    "CaseFieldID": "specClaimTimelineListLabel",
     "PageDisplayOrder": 7,
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "PageID": "HowToAddTimeline",
-    "PageShowCondition": "respondent1ClaimResponsePaymentAdmissionForSpec=\"PAID_LESS_THAN_CLAIMED_AMOUNT\" OR defenceRouteRequired = \"DISPUTES_THE_CLAIM\" OR respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\"",
```

Comment:
Removing this PageShowCondition doesn't make it appear in full admission?

Reply:
I not sure why it was removed, i bring it back, test it and works perfectly